build_tools/setup_helpers/extension.py

```diff
@@ -18,9 +19,7 @@
 _TP_BASE_DIR = _ROOT_DIR / 'third_party'
 _TP_INSTALL_DIR = _TP_BASE_DIR / 'build'
 
-# Temporary fix for building in fbcode
-# at the moment, we have to use external sox in fbcode
-_BUILD_DEPS = not (_ROOT_DIR / '.use_external_sox').exists()
+_BUILD_SOX = 'BUILD_SOX' in os.environ
```

Comment:
I have seen more often a check for whether `BUILD_SOX=1` instead of just the existence. I could imagine someone doing `BUILD_SOX=0` and be surprise that sox is built.

Reply:
So, for this kind of env var, there is no standard of what values are acceptable.
there are really a lot of variations like `0, 1, true, false, on, off, yes, no` etc...
and since this is env var, we should not really raise an error if the value is something unexpected.
I am open in defining values for our case, but my personal preference for binary behavior is just define variable or not.

Reply:
I changed the behavior to expect 0/1 (and some others)